### PR TITLE
Initialize a barebones implementation

### PR DESCRIPTION
**Pull request by HTGAzureX1212**

### CODE REVIEW

File: Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+using DSharpPlus;
+
+using RhythmGamesBot.Utils;
+
 namespace RhythmGamesBot.Core
 {
     public class RhythmGamesBot
     {
+        private static Option<DiscordClient> BotInstance { get; set; }
+        private static DiscordIntents Intents = DiscordIntents.All;
```
Comment (cgytrus) [inline Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs]: I think this can be made a constant?

Comment (HTGAzureX1212) [reply Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs]: Did you mean `Intents`?

Comment (cgytrus) [reply Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs]: yes, i commented on that line.

Comment (HTGAzureX1212) [reply Libraries/RhythmGamesBot.Core/RhythmGamesBot.cs]: Resolved on next commit.

**Review by cgytrus — APPROVED** [review]

**Pull request merged**